iOS/MMIWG2S/ViewControllers/IntroViewController.swift

```diff
@@ -91,7 +87,7 @@ class IntroViewController: UIViewController {
         view.addSubview(labelMMIWInfo)
         view.addSubview(spacerView1)
 
-        let pageControlViews =  [UIView](repeating:UIView(), count:4)
+        let pageControlViews: [UIView] = [UIView(), UIView(), UIView(), UIView()]
```

Comment:
what was wrong with `[UIView](repeating:UIView(), count:4)`?

Reply:
The bars never show up and there is a set of errors complaining about missing constraints. This was my work around. Does it work for you?

Reply:
Sure. I'm really curious, I'd have thought they would do the same thing.

Reply:
Same. I read up on it and it seems to be doing the exact same thing. It was very odd